app/controllers/home_controller.rb

```diff
@@ -5,12 +5,16 @@ class HomeController < ApplicationController
     filters: []
   }.to_json
 
+  # Format for this date is: Month (3 letters) and Year (4 digits) 
+  UPDATED_AT = "Feb 2019"
```

Comment:
@stacytalbot @drewpotter Do you know if this needs to be an official date or can be retrieved by the timestamps of the latest record?

Here is the difference:
If Marine says that the date should be like an official release (of which they are also keeping track of perhaps), then it would make sense to have this manually updated, as the actual month where we perform the import can be different if the data has been given to us on the last day of the previous month for example.

If the above is not necessary, then we can use the latest Evaluation timestamp and format the date accordingly.

@drewpotter If we decide to keep as you did, I would add `.freeze` at the end of the constant definition

Reply:
Done!